### ISSUE TITLE
React Native iOS unable to link @casl

### ISSUE THREAD

**Issue by Ogala** [stalniy/casl#523]
**Describe the bug**
TLDR: React Native iOS unable to link both @casl/ability and @casl/react due to the following warning
```
warn Package @casl/ability has been ignored because it contains invalid configuration. Reason: Package subpath './package.json' is not defined by "exports" in /Users/emmanuelogala/Code/josplayet/node_modules/@casl/ability/package.json
warn Package @casl/react has been ignored because it contains invalid configuration. Reason: Package subpath './package.json' is not defined by "exports" in /Users/emmanuelogala/Code/josplayet/node_modules/@casl/react/package.json
```
I recently upgraded my React and React Native to `17.0.1` and `0.64.2` and @casl stopped working on iOS. It works on Android. During linking, I get the warning above. Due to the failure of linkage - see warning above.

 When I try to use, say `Can`, I get a `Cannot read property 'can' of undefined`

**To Reproduce**

prerequisite: React 17.0.1, react Native 0.64.2
1. `npm install @casl/react @casl/ability`
2. Setup ability as provided in the v5 guide (https://casl.js.org/v5/en/package/casl-react)
3. Use `Can` to check ability

**Expected behavior**
I should be able to use `Can` to check ability.

**Interactive example** (optional, but highly desirable)
provide a link to the example from http://repl.it/, https://codesandbox.io/ or similar, so we can quickly test and provide feedback. Otherwise

**CASL Version**
```
 "@casl/ability": "^5.3.1",
   "@casl/react": "^2.3.0",
```

**Environment**:
```
Nodejs 14-stable 
Xcode 12.5
Build version 12E5234g
```

**Comment by stalniy** [comment]
Did you try to add package.json manually to casl’s repos, in node_modules. Just to see whether it works

**Comment by stalniy** [comment]
any updates on this?

**Comment by Ogala** [comment]
> any updates on this? 

Yeah. I just did. It builds and links. But it still gives me the same "can not defined" error.

**Comment by Todmy** [comment]
I believe you did something wrong. I have created a simple example for the react-native app. You can check it out [here](https://github.com/Todmy/casl-test).

**Comment by stalniy** [comment]
Thanks @Todmy for checking it! 

Close this as I don't see a reason to add package.json into `exports` field for some other library to be able to require it using nodejs `require`. React native should read this file using `fs` and resolve path to it without using `require`. Related RN issue - https://github.com/react-native-community/cli/issues/1168

Otherwise, every library in npm repo that can be used with RN should add package.json into exports, what seems meaningless
